src/test/system_tests/test_lifecycle.js

```diff
@@ -54,9 +54,6 @@ async function main() {
     await commonTests.test_rule_id(Bucket, Key, s3);
     await commonTests.test_filter_size(Bucket, s3);
     await commonTests.test_and_prefix_size(Bucket, Key, s3);
-    await commonTests.test_rule_id_length(Bucket, Key, s3);
```

Comment:
why do you remove it?

Reply:
These tests are written by me previously and are not working with system_tests, so I pushed them to unit_tests together with other tests for proper working.